src/Integration.UnitTests/Binding/CFamilyProjectBinderTests.cs

```diff
@@ -79,31 +79,23 @@ public void Ctor_NullFileSystem_ArgumentNullException()
         }
 
         [TestMethod]
-        [DataRow(true)]
-        [DataRow(false)]
-        public void IsBound_ProjectHasOneLanguage(bool configFileExists)
+        public void IsBindingRequired_ProjectLanguageIsNotSupported_False()
         {
             var projectMock = new ProjectMock("c:\\test.csproj");
             projectMock.SetCSProjectKind();
 
-            solutionRuleSetsInformationProviderMock
-                .Setup(x => 
-                    x.CalculateSolutionSonarQubeRuleSetFilePath("key", Language.CSharp, SonarLintMode.Connected))
-                .Returns("c:\\config-file.txt");
-
-            fileSystemMock
-                .Setup(x => x.File.Exists("c:\\config-file.txt"))
-                .Returns(configFileExists);
-
             var bindingConfiguration = new BindingConfiguration(new BoundSonarQubeProject(new Uri("http://test.com"), "key", "name"),
                 SonarLintMode.Connected, "c:\\");
 
-            var result = testSubject.IsBound(bindingConfiguration, projectMock);
-            result.Should().Be(configFileExists);
+            var result = testSubject.IsBindingRequired(bindingConfiguration, projectMock);
+            result.Should().Be(false);
+
+            solutionRuleSetsInformationProviderMock.VerifyNoOtherCalls();
+            fileSystemMock.VerifyNoOtherCalls();
         }
 
         [TestMethod]
-        public void IsBound_ProjectHasTwoLanguages_OneLanguageHasNoConfigFile_False()
+        public void IsBindingRequired_ProjectHasTwoLanguages_OneLanguageHasNoConfigFile_True()
```

Comment:
It should be possible to write the missing "ProjectHasTwoLanguages_AllConfigFilesExists_False" test now, shouldn't it?